sdk/src/main/java/com/microsoft/hydralab/performance/IBaselineMetrics.java

```diff
@@ -0,0 +1,15 @@
+package com.microsoft.hydralab.performance;
+
+import java.util.LinkedHashMap;
+
+public interface IBaselineMetrics {
+    // Use LinkedHashMap to ensure the order of metrics key-value
+    LinkedHashMap<String, Double> getBaselineMetricsKeyValue();
+
+    SummaryType getSummaryType();
+
+    enum SummaryType {
```

Comment:
We can define more aggregate method and show all the aggregated data in the front end, like 80 percentile, mid number and sum

Reply:
Thanks, will do in the future